src/ui/common/src/components/pages/artifact/id/hook.ts

```diff
@@ -0,0 +1,143 @@
+import { useEffect } from 'react';
```

Comment:
`useArtifact` hook handles redux store retrieval for `artfResults`, `artfContents`. `useArtifactHistory` hook handles redux store retrieval for artf history.